main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -373,10 +373,16 @@ DataTableView.prototype._renderDataTables = function(table, tableHeader) {
     var th = trHead.appendChild(document.createElement("th"));
     $(th).addClass("column-header").attr('title', column.name);
     if (self._collapsedColumnNames.hasOwnProperty(column.name)) {
-      $(th).html("&nbsp;").on('click',function(evt) {
-        delete self._collapsedColumnNames[column.name];
-        self.render();
-      });
+      DOM.bind( 
+        $(th)
+        .attr('title',"Expand this column")
```

Comment:
Oops, here is a problem I oversaw: we'd need this text to be localized, with our `$.i18n` utility.

Reply:
I have changed the text for `$.i18n` utility and I added the text in all the supported languages. Please, tell me if you need anything else 🚀